### ISSUE TITLE
Stops working in about a day, works again if reinstalled

### ISSUE THREAD

**Issue by pystrats** [angristan/wireguard-install#428]
This happens with certain hosting providers, but not the others. Clients connect successfully to the server first, for about a day, but then stop. No matter if you remove old clients and add new ones. If you uninstall Wireguard and then install it again following ./wireguard-install.sh prompts, all works again. 
Any ideas why something like that can happen and how to fix it?

**Comment by HenryCooper86** [comment]
Similar issue, on AWS's Lightsail

**Comment by pystrats** [comment]
Yes, this happens on all AWS EC2 instances for me as well. And I encountered the same issue with other, smaller hosting providers.

**Comment by turbozapekanka** [comment]
I noticed the same with OCI hosted VMs, and only for some clients. No handshake and no internet connectivity.

**Comment by g40** [comment]
Anyone got any more data for this? I have exactly the same problem on a Ubuntu Linode.

**Comment by sameeraksc** [comment]
we are also having this for only some clients. no internet connectivity for them. when I restarted the wire guard, its started to work for them.
